frontend/styles/var/breakpoints.scss

```diff
@@ -2,3 +2,4 @@ $break-1600: 1600px;
 $break-1400: 1400px;
 $break-990: 990px;
 $break-768: 768px;
+$tooltip-break-1000: 1000px; // Prevents horizontal scrolling off viewport
```

Comment:
This seems like a global breakpoint we are adding for a single specific fix. do you think we will use this breakpoint in another location? If not i'd just assign `@media (max-width: 1000px)` in the `_styles.scss` and removing it here so this file only contains reused breakpoints.

Reply:
My hunch is, as we've continued to add tooltips, we will run into this issue anytime a long text tooltip is in the last column of a table. Since it is single use, we might not want to define a global breakpoint, but I also don't want someone to have to try to reinvent the wheel if they run into this again.

Reply:
ok cool seems fine how it is then ✅